narwhal/config/src/lib.rs

```diff
@@ -381,8 +381,8 @@ impl Default for Parameters {
         Self {
             header_num_of_batches_threshold: 32,
             max_header_num_of_batches: 1000,
-            max_header_delay: Duration::from_millis(100),
-            min_header_delay: Duration::from_millis(100),
+            max_header_delay: Duration::from_millis(2000),
```

Comment:
Can you call `default_max_header_delay()` and `default_min_header_delay()` instead? Maybe update other fields to call the default value function as well.